core/src/main/scala/kafka/tools/StreamsResetter.java

```diff
@@ -655,7 +655,7 @@ public void doDelete(final List<String> topicsToDelete,
                          final Admin adminClient) {
         boolean hasDeleteErrors = false;
         final DeleteTopicsResult deleteTopicsResult = adminClient.deleteTopics(topicsToDelete);
-        final Map<String, KafkaFuture<Void>> results = deleteTopicsResult.values();
+        final Map<String, KafkaFuture<Void>> results = deleteTopicsResult.topicNameValues();
```

Comment:
Deprecated via https://github.com/apache/kafka/commit/cee2e975d124da0a2bdc0065a3172ce31e036fa0

Could not find the corresponding KIP... \cc @jolshan @hachikuji

Reply:
Thanks! Can we cross link KIP wiki page and Jiras to make it easier to find?

Reply:
I believe the jira was under the kip's main jira, but I can make the link clearer.

Reply:
Thanks!

It's also best, to mention a KIP in the commit message. I guess @hachikuji just forgot to add it?